ampd/src/grpc/ampd.rs

```diff
@@ -0,0 +1,37 @@
+use std::pin::Pin;
+
+use async_trait::async_trait;
+use futures::Stream;
+use tonic::{Request, Response, Status};
+
+pub mod proto {
+    tonic::include_proto!("ampd");
+}
+
+pub struct Ampd {}
+
+impl Ampd {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+#[async_trait]
+impl proto::ampd_server::Ampd for Ampd {
+    type SubscribeStream =
```

Comment:
```suggestion
    type SubscriptionStream =
```

Reply:
The trait is auto-generated code that I cannot change without changing the service method to `Subscription`.

Reply:
ah